common/scripted_triggers/00_cultural_triggers.txt

```diff
@@ -287,8 +316,11 @@ culture_has_heavy_cavalry_maa = {
 		# has_innovation = innovation_valets #Gendarme - Later Era MAA
 
 		# Warcraft
-		# TODO
-		# Add mod MaAs here
+		has_cultural_parameter = unlock_maa_kodo_rider
+		has_cultural_parameter = unlock_maa_mushan_rider
+		has_cultural_parameter = unlock_maa_turtle_rider
+		has_cultural_parameter = unlock_maa_war_dinosaur
```

Comment:
If I remember right, I moved them to the another unit category (thus uniting them with the elephants).
But anyway, I don't get why it's on this branch because I can't find where it's set.

Reply:
Where did you move them?

Reply:
> Where did you move them?

`1.5.0-preview-adaptation`

Reply:
Merged rn with 1.5.0 and only war dinos are elephant_cav. Did you want all 4 of these as elphant_cav?

Reply:
> Merged rn with 1.5.0 and only war dinos are elephant_cav. Did you want all 4 of these as elphant_cav?

Yes, all "giant" cavalry go there.